### ISSUE TITLE
TypeError when including the standalone file

### ISSUE THREAD

**Issue by dcominottim** [atomiks/tippyjs#73]
When including the standalone file from version 1.1.1, I am seeing this error on Chrome Dev Tools:

```
tippy.standalone.js:1 Uncaught TypeError: Cannot use 'in' operator to search for 'default' in undefined
    at tippy.standalone.js:1
```

**Comment by atomiks** [comment]
Seems like the bundler added a line that shouldn't be there, no idea why. Will fix it in a sec.

**Comment by atomiks** [comment]
fixed in `1.1.3`

**Comment by SensationSama** [comment]
Hey I'm not sure if you also manage `@tippyjs/react` but that package is still using `"rollup": "^1.1.2",` so the error persists.
